### PR TITLE
Field changes

### PR DESCRIPTION
**Pull request by paulshealy1**
# PR into Azure/azureml-v2-preview

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

-

Fixes #

### CODE REVIEW

File: sdk2-src/tests/test_configs/sweep_job_test.yaml [hunk 1]
```diff
@@ -24,10 +24,9 @@ trial:
     name: /subscriptions/5f08d643-1910-4a38-a7c7-84a39d4f42e0/resourceGroups/sdk_vnext_cli/providers/Microsoft.MachineLearningServices/sdk_vnext_cli/providers/Microsoft.MachineLearningServices/workspaces/sdk_vnext_cli/Environments/AzureML-Minimal
   compute:
     target: azureml:aBadCompute
-limits:
-  max_total_runs: 100
-  max_concurrent_runs: 10
-  max_duration_minutes: 10000
+max_total_runs: 100
+max_concurrent_runs: 10
+max_duration_minutes: 10000
```
Comment (rihorn2) [inline sdk2-src/tests/test_configs/sweep_job_test.yaml:29]: I believe this is just called timeout_minutes

File: sdk2-src/tests/test_configs/sweep_job_test.yaml [hunk 2]
```diff
@@ -24,10 +24,9 @@ trial:
     name: /subscriptions/5f08d643-1910-4a38-a7c7-84a39d4f42e0/resourceGroups/sdk_vnext_cli/providers/Microsoft.MachineLearningServices/sdk_vnext_cli/providers/Microsoft.MachineLearningServices/workspaces/sdk_vnext_cli/Environments/AzureML-Minimal
   compute:
     target: azureml:aBadCompute
-limits:
-  max_total_runs: 100
-  max_concurrent_runs: 10
-  max_duration_minutes: 10000
+max_total_runs: 100
```
Comment (rihorn2) [inline sdk2-src/tests/test_configs/sweep_job_test.yaml:27]: this is max_total_trials now,
and max_concurrent_trials

File: sdk2-src/samples/yaml/from_command-SweepJob.yaml [hunk 3]
```diff
@@ -22,10 +22,9 @@ trial:
     value:
       data: azureml:/subscriptions/5f08d643-1910-4a38-a7c7-84a39d4f42e0/resourceGroups/sdk_vnext_cli/providers/Microsoft.MachineLearningServices/workspaces/sdk_vnext_cli/data/vnext_test/versions/1
       mode: Mount
-limits:
-  max_total_runs: 100
-  max_concurrent_runs: 10
-  max_duration_minutes: 10000
+max_total_runs: 100
```
Comment (rihorn2) [inline sdk2-src/samples/yaml/from_command-SweepJob.yaml:25]: Same name changes as in the bottom file

File: sdk2-src/samples/yaml/model.azml-Model.yaml [hunk 4]
```diff
@@ -2,7 +2,6 @@
 name: my-model
 asset_path: ./model
```
Comment (mx-iao) [inline sdk2-src/samples/yaml/model.azml-Model.yaml:3]: should this be "local_path" instead of "asset_path"

**Review by mx-iao — APPROVED** [review]

**Pull request merged**